### PR TITLE
Checkout

### PR DESCRIPTION
**Pull request by yieknee**

### CODE REVIEW

File: app/models/order.rb
Comment (cojenco) [inline app/models/order.rb:9]: cc_number length validation should be 16? Once we merge, let's check if the tests and orders#show need to be revised.

File: app/models/order.rb
Comment (cojenco) [inline app/models/order.rb]: It seems like "quantity" is missing in this calculation. Please note that I made a #subtotal instance method in the OrderItem model, which we should be able to incorporate into this method. We will need to merge both PR's first👍

**Review by cojenco — APPROVED** [review]
LGTM!!!🏆 Once PR#11 and PR#12 are merged, we can incorporate order_items#subtotal instance method into the order_total method here. And tests.

**Review by aecombs — CHANGES_REQUESTED** [review]
Code looks good -- echoing Cathy for cc_number validations and order_total method, and also add tests for controller actions and model methods/validations/relations

File: app/models/order.rb
Comment (yieknee) [reply app/models/order.rb]: i've made the change

**Pull request merged**
